src/main/java/Suv.java

```diff
@@ -0,0 +1,94 @@
+                    int index = i + 1;
+                    System.out.println(" " + index + "." + tasks[i]);
+                }
+                System.out.println("____________________________________________________________\n"
+                );
+            } else if (input.contains("unmark")){
+                int n = Integer.parseInt(input.split(" ")[1]);
+                tasks[n - 1].unMark();
+                System.out.println("____________________________________________________________\n" +
+                        " OK, I've marked this task as not done yet:\n" + "   [ ] " + tasks[n - 1].getDescription() +
+                        "\n____________________________________________________________\n"
+                );
+            } else if (input.contains("mark")){
+                int n = Integer.parseInt(input.split(" ")[1]);
+                Task currentTask = tasks[n - 1];
+                currentTask.mark();
+                System.out.println("____________________________________________________________\n" +
+                        " Nice! I've marked this task as done:\n" + "   [X] " + currentTask.getDescription() +
+                        "\n____________________________________________________________\n"
+                );
+            } else if(input.contains("todo")){
+                Todo newTask = new Todo(input.substring(TODO_KEYWORD_END_INDEX).trim());
+
+                tasks[taskIndex++] = newTask;
+                String helper = (taskIndex > 1) ? "s " : " ";
+                System.out.println("____________________________________________________________\n" +
+                        " Got it. I've added this task:\n" + "  " + newTask +
+                        "\n Now you have " + Integer.toString((taskIndex)) +" task" + helper + "in the list." +
+                        "\n____________________________________________________________\n"
+                );
+            }else if(input.contains("deadline")){
+                String by = input.split("/by")[1].trim();
+                String description = input.split("/by")[0].substring(DEADLINE_KEYWORD_END_INDEX).trim();
+
+                Deadline newTask = new Deadline(description, by);
+                tasks[taskIndex++] = newTask;
+                String helper = (taskIndex > 1) ? "s " : " ";
+                System.out.println("____________________________________________________________\n" +
+                        " Got it. I've added this task:\n" + "  " + newTask +
+                        "\n Now you have " + Integer.toString((taskIndex)) +" task" + helper +"in the list." +
+                        "\n____________________________________________________________\n"
+                );
+            } else if(input.contains("event")){
+                String from = input.split("/")[1].trim().substring(FROM_KEYWORD_END_INDEX).trim();
+                String to = input.split("/")[2].trim().substring(TO_KEYWORD_END_INDEX).trim();
```

Comment:
perhaps we could abstract the string processing into a parser class with descriptive methods for better readability in the main file